etc/profile.d/git-sdk.sh

```diff
@@ -126,7 +125,7 @@ sdk () {
 		case "$2" in
 		git)
 			sdk init git &&
-			make -C "$src_dir" -j$(nproc) DEVELOPER=1
+			make -C "$src_dir" -j$(nproc) 
```

Comment:
How about leaving this as-is? If you call `sdk build git`, we are building using GCC, after all...

Reply:
But, do we actually want to build with `DEVELOPER=1`?